_data/binford2k.json

```diff
@@ -0,0 +1 @@
+{"login":"binford2k","lat": 45.505556, "lng": 122.655}
```

Comment:
I feel like a minus sign is missing before the longitude :stuck_out_tongue_winking_eye: